src/genie/libs/parser/hp_comware/tests/DisplayInterfacesBrief/cli/equal/golden_output_output.txt

```diff
@@ -0,0 +1,22 @@
+The brief information of interface(s) under route mode:
+Link: ADM - administratively down; Stby - standby
+Protocol: (s) - spoofing
+Interface            Link Protocol Main IP         Description
+M-E1/0/0/0           DOWN DOWN     10.0.0.1
```

Comment:
`M-E1/0/0/0` or `BAGG1` is kind of interface short name? like in IOSXE `Lo0`. (=`Loopback0`)
In genie parser for XE/XR/NX, we usually convert short name to full name. so that it will be easy to compare/check same interface between parsers output.

If short name, we might want to do the same in this HP parser.

Reply:
Sure, but in this case perhaps the best place to translate is in the genie.libs.parser.utils.common.Commom.convert_intf_name module, what do you think?
I could modify it to support these abbreviations

Reply:
sorry, I didn't provide the pointer to `convert_intf_name`. but you found it! yes, that's the one we are using to convert. yes, please go for supporting for Comware interfaces.

Reply:
Support added on  0f1c0a8